src/arch/mpi/machine.C

```diff
@@ -1787,17 +1786,16 @@ void LrtsPreCommonInit(int everReturn) {
     CpvAccess(crashedRankHdr) = NULL;
     CpvAccess(crashedRankPtr) = NULL;
 #endif
-}
-
-void LrtsPostCommonInit(int everReturn) {
```

Comment:
Any insight on why these were here before, or what functionally the difference is between things that can go inside LrtsInit and thing that should go in LrtsPostCommonInit?

Reply:
They were added in commit ff22b81b1c2058cd467e75071ac02fb7ebe6a19c, when these variables were turned into `Cpv`s. I don't see any reason to keep them there, since `LrtsPreCommonInit` has other `Cpv` initializations. If they are not moved, attempting to use Converse communication during `ConverseCommonInit` crashes due to null pointer deference on them.